app/Http/Controllers/WebhookController.php

```diff
@@ -62,9 +62,15 @@ public function store(WebhookCreateRequest $request)
             $webhook = $this->webhookRepository->create($data);
 
             return redirect()->route('webhooks.edit', $webhook)
-                ->with('messageSuccess', 'This webhook successfully created');
+                ->with('messageSuccess', [
+                    'status' => 'Create success',
+                    'message' => 'This webhook successfully created'
+                ]);
         } catch (QueryException $exception) {
-            return redirect()->back()->with('messageFail', 'Create failed. Something went wrong')->withInput();
+            return redirect()->back()->with('messageFail', [
+                'status' => 'Create success',
+                'message' => 'Create failed. Something went wrong'
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Arrays.ArrayDeclaration.NoComma</br>**Severity**: error</br>**File**: app/Http/Controllers/WebhookController.php</br>**Line**: 72</br>Each line in an array declaration must end in a comma